bucket/teracopy-np.json

```
@@ -4,17 +4,16 @@
     "homepage": "https://www.codesector.com/teracopy",
     "license": "Freeware",
     "url": "http://www.codesector.com/files/teracopy.exe",
-    "hash": "9dfa57c6dd935f72e7e8254146cd143528fe962d2c2f518c2d6511d4d1150a3a",
+    "hash": "e2dfc14c3b143e1d8d98305bf118651e433d053a437e13f4f5c463bede692ac5",
     "installer": {
         "file": "teracopy.exe",
         "args": [
-            "/verysilent",
-            "/nocancel",
-            "/norestart"
+        	"/qn",
+        	"/passive"
         ]
     },
     "uninstaller": {
-        "script": "& \"$env:PROGRAMFILES\\TeraCopy\\unins000.exe\" /verysilent /nocancel /norestart"
+        "script": "& \"$env:PROGRAMFILES\\TeraCopy\\unins000.exe\" /qn /passive"
```

```suggestion
        "script": "Invoke-ExternalCommand msiexec -ArgumentList @('/x', '{C38F85E6-6E93-4963-A29E-B871C8A28D13}', '/qn', '/norestart') -RunAs | Out-Null"
```

`unins000.exe` does not exist in the new version.

The uninstall command for *TeraCopy* (saved in registry `HKLM:\Software\Microsoft\CurrentVersion\Uninstall` is:
`msiexec /i {C38F85E6-6E93-4963-A29E-B871C8A28D13}`